lib/individual/health.dart

```diff
@@ -1,182 +1,220 @@
+
+
+
 import 'package:flutter/material.dart';
+import 'package:cura/shared/widgets/gradient_background.dart';
+import 'package:cura/main.dart';
 import 'package:flutter_screenutil/flutter_screenutil.dart';
 import 'package:searchfield/searchfield.dart';
+// import 'package:searchfield/searchfield.dart';
 
-class HealthPage extends StatefulWidget {
-  static const String routeName = '/HealthPage';
```

Comment:
And please don't remove the routeName variable, as it is of paramount importance in the routing.